bundles/action/org.openhab.action.pushover/README.md

```diff
@@ -4,46 +4,49 @@ The Pushover action service allows you to notify mobile devices of a message usi
 
 ## Configuration
 
-You may define default values for parameters to action calls in the file `services/pushover.cfg`.  None of the configuration parameters are required as you can specify required configuration items in the action call, but you must at least provide an _API Token_, _User/Group Key_ and a _Message_ in some manner before a message can be pushed.
-
-| Property | Default | Required | Description |
-|----------|---------|:--------:|-------------|
-| defaultTimeout | 10000 | No   | Timeout in milliseconds for the connection to pushover.net |
-| defaultToken | | if using action call without `apiToken` parameter | Pushover [API token](https://pushover.net/api) to send to devices |
-| defaultUser | | if using action call without `userKey` parameter | Pushover User or Group key (not e-mail address) of your user (or you) to send to devices. |
-| defaultTitle | openHAB | No | Application title for the notification |
-| defaultPriority | 0 | No | Priority of the notification, from -2 (low priority) to 2 (high priority) |
-| defaultUrl | | No | URL to attach to the message if not specified in the command. This can be used to trigger actions on the device. |
-| defaultUrlTitle | | No | URL title to attach to the message if not specified in the command. This can be used to trigger actions on the device. |
-| defaultRetry | 300 | No | When priority is 2 (high priority), how often (in seconds) should messages be resent |
-| defaultExpire | 3600 | No | When priority is 2 (high priority), how long (in seconds) to continue resending messages until acknowledged |
+You may define default values for parameters to action calls in the file `services/pushover.cfg`.
+None of the configuration parameters are required as you can specify required configuration items in the action call, but you must at least provide an *API Token*, *User/Group Key* and a *Message* in some manner before a message can be pushed.
+
+| Property        | Default | Required                                          | Description                                                                                                            |
+|-----------------|---------|-:-:-----------------------------------------------|------------------------------------------------------------------------------------------------------------------------|

```

Comment:
This is broken.

Reply:
Hi @9037568.  I believe my latest commit fixes this.  Please look, and if okay, please merge.

Reply:
Sorry, the last commit doesn't appear to have changed anything.